# The Angle2D message represents the horizontal and vertical angle on 2D plane

float64 x
float64 y
